src/main/kotlin/it/pagopa/wallet/exceptionhandler/ExceptionHandler.kt

```diff
@@ -43,6 +40,13 @@ class ExceptionHandler {
     /** ApiError exception handler */
     @ExceptionHandler(ApiError::class)
     fun handleException(e: ApiError): ResponseEntity<ProblemJsonDto> {
+        if (e is WalletConflictStatusException) {
+            logger.warn(
+                "Conflict on wallet [{}] with status [{}]",
+                e.walletId.value.toString(),
+                e.walletStatusDto.value
+            )
+        }
```

Comment:
can we remove this and change `WalletConflictStatusException` to match `Conflict on wallet [{}] with status [{}]`?

Reply:
Solved by https://github.com/pagopa/pagopa-wallet-service/pull/150/commits/2043d528e69de69d75a670d36b48a6838475b0b3